pyne/r2s.py

```diff
@@ -180,8 +182,18 @@ def total_photon_source_intensity(m, tag_name):
 
     sd_tag = m.mesh.getTagHandle(tag_name)
     intensity = 0.
-    for idx, _, ve in m:
-        vol = m.elem_volume(ve)
-        ve_data = sd_tag[ve]
-        intensity += vol*np.sum(ve_data)
+    if sub_voxel:
```

Comment:
Look at reorganizing this conditional and loop

Reply:
@gonuke I found that the `max_num_cells` can not be moved outside the conditional statement because the mesh doesn't have a `cell_fracs` tag under r2s mode.